docs/administration/moving-your-octopus/move-the-home-directory.md

```diff
@@ -0,0 +1,138 @@
+---
+title: Moving the Octopus Home Directory
+description: Information on how to move any of the Octopus Server folders.
+position: 400
+---
+
+If you need to move any of the folders used by the Octopus Server you can follow the instructions on this page to move individual folders and reconfigure the Octopus Server to use the new folder locations.
+
+## Move Octopus Home folder {#MovingOctopusServerfolders-OctopusHomeMoveOctopusHomefolder}
+
+If you need to move the Octopus Home folder, you can do that using the command-line as described below:
+
+**Usage**
+
+```powershell
+Usage: Octopus.Server configure [<options>]
+```
+
+Where `[<options>]` is:
+
+**Options**
+
+```powershell
+      --instance=VALUE       Name of the instance to use
+      --home=VALUE           Home directory
+
+Or one of the common options:
+
+      --console              Don't attempt to run as a service, even if the
+                               user is non-interactive
+      --nologo               Don't print title or version information
+```
+
+A PowerShell script showing the steps is set out below. You need to change the variables to match your Octopus installation, and you may wish to run each step separately to deal with any issues like locked files.
+
+```powershell
+$oldHome = "C:\Octopus"
+$newHome = "C:\YourNewHomeDir"
+$octopus = "C:\Program Files\Octopus Deploy\Octopus\Octopus.Server.exe"
+$newConfig = $newHome + "\OctopusServer.config"
+& "$octopus" service --stop
+mv $oldHome $newHome
+ 
+&"$octopus" delete-instance --instance=OctopusServer
+&"$octopus" create-instance --instance=OctopusServer --config=$newConfig
+
+& "$octopus" configure --home="$newHome"
+& "$octopus" service --start
+```
+
+## Move other Octopus Server folders {#MovingOctopusServerfolders-MoveotherOctopusServerfolders}
+
+If you need to move other folders than the Octopus Home folder, you can do that using the command-line as described below
+
+**Usage**
+
+```powershell
+Octopus.Server path [<options>]
+```
+
+Where `[<options>]` is any of:
+
+**Options**
+
+```powershell
+      --instance=VALUE       Name of the instance to use
+      --nugetRepository=VALUE
+                             Set the package path for the built-in NuGet 
+                               repository.
+      --artifacts=VALUE      Set the path where artifacts are stored.
+      --taskLogs=VALUE       Set the path where task logs are stored.
+
+Or one of the common options: 
+      --console              Don't attempt to run as a service, even if the 
+                               user is non-interactive
+      --nologo               Don't print title or version information 
+```
+
+## Move NuGet repository folder {#MovingOctopusServerfolders-NuGetRepositoryMoveNuGetrepositoryfolder}
+
+A PowerShell script showing the steps is set out below. You need to change the variables to match your Octopus installation,
+and you may wish to run each step separately to deal with any issues like locked files. 
```

Comment:
Are newlines intended here? Markdown will interpret them as new paragraphs, so if this is meant to be one paragraph, remove newlines here.